Exercise_1/index.html

```diff
@@ -0,0 +1,128 @@
+<!doctype html>
+<html lang="pl">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+<table width="100%" border="1" cellpadding="5px">
+    <thead>
+    <tr>
+        <th colspan="3" height="100px">
+            <h1>Guide to the City</h1>
+        </th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td valign="top" width="150px">
+            <h2 align="center">
+                Navigation
+            </h2>
+            <ul>
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Hotels</a></li>
+                <li><a href="#">Restaurants</a></li>
+                <li><a href="#">Contacts</a></li>
+            </ul>
+        </td>
+        <td valign="top">
+            <h2 align="center">Kraków</h2>
+            <article>
+                Kraków <sup>1</sup> is simply dummy text of the printing and typesetting industry.
+                Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer
+                took a galley of type and scrambled it to make a type specimen book. It has survived not only five
+                centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was
+                popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and
+                more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer
+                took a galley of type and scrambled it to make a type specimen book. It has survived not only five
+                centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was
+                popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and
+                more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+            </article>
+            <img src="img/images.png"
```

Comment:
And here the mess starts. You have an article and it is ok, but your image is out of anything. It is not good. It should be a part of the article or, if not, it should be wrapped in `<aside>`. Anyway avoid using `<img>` tag not wrapped.